cli/cmd/vuln_container.go

```diff
@@ -520,26 +544,40 @@ func vulContainerImagePackagesToTable(image *api.VulnContainerImage) [][]string
 		}
 	}
 
+	if vulFiltersEnabled() {
+		filteredOutput = fmt.Sprintf("%v of %v packages showing \n", len(out), packagesCount)
```

Comment:
I think the count of packages is wrong. use this command inside our tech-ally account:
```
lacework vuln ctr show sha256:4a69362cb4a07631be9d5688e81befa5a39e02d117411242c83617238393565e --severity high --packages
```
You will see:
```
  CVE COUNT   SEVERITY   PACKAGE   CURRENT VERSION   FIX VERSION
------------+----------+---------+-----------------+--------------
  2           High       curl      7.69.1-r1         7.69.1-r3
  1           High       curl      7.69.1-r1         7.69.1-r2
2 of 1 packages showing
```